src/resources/elements/optionfield.js

```diff
@@ -189,6 +189,33 @@ export class Optionfield extends Field {
   }
 
   /**
+   * Get the label(s) of the currently chosen option(s)
+   * @return {String|String[]} The label or labels depending on the format of
+   *                           this Optionfield.
+   */
+  getChoiceLabel() {
+    if (this.format === 'dropdown') {
```

Comment:
Remember plain language comments. This code is very readable, but is still JavaScript. JS syntax has noise ( '===', '{' '}' ...) that a plain language comment doesn't need to contain.